test/test_unistd.rs

```diff
@@ -625,10 +625,11 @@ fn test_acct() {
     acct::disable().unwrap();
 }
 
+#[cfg(not(target_os = "hurd"))]
```

Comment:
Better to build the test, but ignore it:
```suggestion
#[cfg_addr(target_os = "hurd", ignore)]
```

Reply:
ok, done so!

Reply:
Seems that this requested change has been reverted accidentally

Reply:
oops, sorry, indeed, fixed back